lib/paf.ml

```diff
@@ -303,30 +303,32 @@ type impl = Runtime : 'conn runtime * 'conn -> impl
 type 't service =
   | Service : {
       accept : 't -> ('flow, ([> `Closed ] as 'error)) result Lwt.t;
-      connection : 'flow -> (Mimic.flow * impl, 'error) result Lwt.t;
+      connect : 'flow -> ('connected_flow, ([> `Closed ] as 'error)) result Lwt.t;
```

Comment:
It's probably the best to rename it `handshake`, this is what we want to really operate.